Makefile

```diff
@@ -21,13 +21,6 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-# Tag parameters
-MAJOR=1
-MINOR=0
-PATCH=0
-NOTES=-beta
-TAGMSG="CSI Spec 1.6"
-
```

Comment:
Could you check if these lines in docker.mk are required now ?
https://github.com/dell/csi-powermax/blob/2926bca9fe072febbc2b865ec58b63deacbabb1d/docker.mk#L19-L27